rfc/wip-tform/index.md

```diff
@@ -0,0 +1,681 @@
+- MAY contain the field "discrete". The value MUST be a boolean, and is `true` if the axis represents a discrete dimension.
+- SHOULD contain the field "unit" to specify the physical unit of this dimension. The value SHOULD be one of the following strings, which are valid units according to UDUNITS-2.
+    - Units for "space" axes: 'angstrom', 'attometer', 'centimeter', 'decimeter', 'exameter', 'femtometer', 'foot', 'gigameter', 'hectometer', 'inch', 'kilometer', 'megameter', 'meter', 'micrometer', 'mile', 'millimeter', 'nanometer', 'parsec', 'petameter', 'picometer', 'terameter', 'yard', 'yoctometer', 'yottameter', 'zeptometer', 'zettameter'
+    - Units for "time" axes: 'attosecond', 'centisecond', 'day', 'decisecond', 'exasecond', 'femtosecond', 'gigasecond', 'hectosecond', 'hour', 'kilosecond', 'megasecond', 'microsecond', 'millisecond', 'minute', 'nanosecond', 'petasecond', 'picosecond', 'second', 'terasecond', 'yoctosecond', 'yottasecond', 'zeptosecond', 'zettasecond'
+- MAY contain the field "longName". The value MUST be a string, and can provide a longer name or description of an axis and its properties.
+
+If part of multiscales metadata, the length of "axes" MUST be equal to the number of dimensions of the arrays that contain the image data.
+
+Arrays are often thought of as containing discrete samples along the continuous variable. Axes representing space and time are
+usually continuous, meaning they can be indexed by real-valued (floating point) numbers whereas discrete axes may be indexed
+only by integers. Arrays are inherently discrete (see Array coordinate systems, below) , but values "in between" discrete
+coordinates can be retreived using an *interpolation* method.  If an axis is continuous (`"discrete" : false`), it indicates
+that indexing with continuous values is meaningful, and that interpolation may be performed along that axis. Interpolation may
+be performed jointly across axes with the same `type`, and interpolation should not be performed for discrete axes or jointly
+across axes with differing `type`s.  Other non-continuous variables and axis types are also usually discrete, such as
+`channel`s, `coordinate`s, and `displacement`s.
+
+Note: The most common methods for interpolation are "nearest neighbor", "linear", "cubic", and "windowed sinc". Here, we refer
+to any method that obtains values at real valued coordinates using discrete samples as an "interpolator". As such, label images
+may be interpolated using "nearest neighbor" to obtain labels at points along the continuum.
+
+
+### Array coordinate systems
+
+Every array has a default coordinate system whose parameters need not be explicitly defined.  Its name is the path to the array
+in the container, its axes have `"type":"array"`, are unitless, and have default "name"s. The ith axis has `"name":"dim_i"`
```

Comment:
Why make the default name require information about where the array is stored in the container? This means that accessing the default coordinate system requires accessing the storage layer for the array, which rules out anything that acts exclusively on metadata. 

Perhaps it would be simpler to reserve a string like "default" for the default coordinate system.

Reply:
Because the coordinate systems of different arrays need to be different by default.